include/NovelRT/Experimental/Input/Glfw/GlfwInputService.h

```diff
@@ -0,0 +1,45 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_EXPERIMENTAL_INPUT_GLFW_GLFWINPUTSERVICE_H
+#define NOVELRT_EXPERIMENTAL_INPUT_GLFW_GLFWINPUTSERVICE_H
+
+#ifndef NOVELRT_EXPERIMENTAL_INPUT_GLFW_H
+#error NovelRT does not support including types explicitly by default. Please include Input.Glfw.h instead for the Input::Glfw namespace subset.
+#endif
+
+namespace NovelRT::Experimental::Input::Glfw
+{
+    class GlfwInputService final : public Input::IInputService
```

Comment:
I think this should be `GlfwInputDevice` as that's more in line with our LLAPI-style conventions going forward.